### PR TITLE
feat(TDI-40351): Add alternative schema name

### PR DESCRIPTION
**Pull request by chmyga**
* Add alternative schema name to output component

* Delete unnecessary classes

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/94/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/94/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/109/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/109/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/110/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/110/)

File: components/components-snowflake/components-snowflake-definition/src/main/java/org/talend/components/snowflake/SnowflakeConnectionProperties.java [hunk 1]
```diff
@@ -92,6 +92,12 @@
 
     public ComponentReferenceProperties<SnowflakeConnectionProperties> referencedComponent = new ComponentReferenceProperties<>("referencedComponent", TSnowflakeConnectionDefinition.COMPONENT_NAME);
 
+    public Property<Boolean> useAlternativeSchema = newBoolean("useAlternativeSchema", false);
+
+    public Property<String> alternativeSchemaName = newString("alternativeSchemaName").setRequired(); //$NON-NLS-1$
+
+    private boolean withAlternativeSchema;
```
Comment (dmytro-sylaiev) [inline components/components-snowflake/components-snowflake-definition/src/main/java/org/talend/components/snowflake/SnowflakeConnectionProperties.java]: Could you please use more appropriate name for this variable? Current name is kind of confusing
Thank you

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/138/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/138/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Adchmyga%2FTDI-40351_use_alternative_schema)

**Comment by chmyga** [comment]
This PR is not required any more. Closing it.

**Pull request closed without merging**